.github/workflows/ci.yml

```diff
@@ -0,0 +1,17 @@
+name: run-tests
+
+on:
+  push:
+
+jobs:
+  test:
+    strategy:
+      matrix:
+        os: [ubuntu-latest, windows-latest]
+    runs-on: ${{ matrix.os }}
+
+    steps:
+      - uses: actions/checkout@v3
+
+      - name: Run Maven tests
```

Comment:
We could use verify which also runs static code analyzer and spotbugs etc

Reply:
Done.